helm/kube-prometheus/charts/exporter-kube-api/templates/service.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: v1
+kind: Service
+metadata:
+  labels:
+    app: {{ template "name" . }}
+    component: kube-api
+    heritage: {{ .Release.Service }}
+    release: {{ .Release.Name }}
+    chart: {{ .Chart.Name }}-{{ .Chart.Version | replace "+" "_" }}
+  name: {{ template "fullname" . }}
+  namespace: kube-system
+spec:
+  clusterIP: None
+  ports:
+    - name: https-metrics
+      port: 443
+      protocol: TCP
+      targetPort: 443
```

Comment:
@bradbeam hm... the case is that api by default exposed on 443. 
Do any one use other ports for api?